tests/data/mc4_readrows.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
```

Comment:
This is important because without this line, the interpreter may not be able to correctly interpret non-ASCII characters in the source code, leading to errors or unexpected behavior. Therefore, it's generally a good practice to include this line in all Python files, especially if they contain non-ASCII characters.

like '๮|๞|๨|๡|๷|๻|๫'  in the code. krub.

Reply:
I get it krub. But what I mean is we may not need the whole file. 

If my understanding is correct, this file is the file you write to view each line of dataset. And if it does, I think we don't need this file in repository krub. 

Please correct me if I'm wrong